EIPS/eip-2124.md

```diff
@@ -0,0 +1,264 @@
+---
+eip: 2124
+title: Zero RTT netsplit on chain mismatch
```

Comment:
Actually, perhaps the title could be improved to reflect the proposal better.

Reply:
Suggestions? :P

Reply:
Dunno, perhaps: "ENR extension: Cheap chain compatibility check" ?

Reply:
`0-RTT chain compatibility check`?

Reply:
I guess that works. Are you not in favour having "ENR extension" as part of the title? i.e. do you think this EIP could change so that it won't be an ENR extension anymore?

Reply:
Updated it to "Zero RTT chain compatibility check"

Originally I wanted this to employ the same mechanism in both ENR and the eth subprotocol. If we upgrade `eth` however, we might as well add some more interesting capabilities, so for now it's better to keep this EIP smaller.

I'm not a fan of tagging this explicitly with ENR. The next generation discovery protocol will have ENRs as native building blocks anyway and this particular EIP is not really an optional extension (apart from backward compatibility if it's missing) since it can make the network significantly more robust.

Reply:
Well the title can also be updated later, but now I think it is more self describing. Can you update the discussion issue too?

Reply:
Fixed.